analyzer/src/analyze/attribute.py

```diff
@@ -25,6 +25,7 @@ def __init__(
     ):
         self.path = path
         self.input_groups = input_groups or []
+        self.type = numerical_types_map.get(definition_id, str)
```

Comment:
lol we didn't even have this

Reply:
I didn't wrote an issue for this 🙂
Thanks Jason for your help!

Reply:
https://github.com/arkhn/fhir-river/issues/155